src/Chapter2.hs

```diff
@@ -349,8 +354,8 @@ Implement a function that returns only the first half of a given list.
 "b"
 -}
 -- PUT THE FUNCTION TYPE IN HERE
-firstHalf l = error "firstHalf: Not implemented!"
-
+firstHalf :: [a] -> [a]
+firstHalf l = fst $ splitAt (length l `div` 2) l
```

Comment:
Interesting solution 🎨 

You also can use `take` to take only `length l `div` 2` elements 👍🏼